### PR TITLE
Release/v3.0.0

### PR DESCRIPTION
**Pull request by BenCShutt**
## Description
Fix breaking changes after bumping `ThunderBasics`.
Specifically the following properties have been removed on the `UIView`:
- `borderColor`
- `borderWidth`
- `cornerRadius`
- `shadowColor`
- `shadowOpacity`
- `shadowRadius`

Also the following classes have been removed:
- `TSCTextView`
- `TSCView`
- `TSCImageView`

## Related Issue
See `ThunderBasics` PR here:
https://github.com/3sidedcube/ThunderBasics/pull/56

## Motivation and Context
Resolve conflicts with `UIView` properties and unnecessary classes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-github "3sidedcube/ThunderTable" == 1.5.0
-github "3sidedcube/ThunderCollection" == 1.2.5
-github "3sidedcube/ThunderBasics" == 1.8.0
-github "3sidedcube/ThunderRequest" == 2.2.2
-github "3sidedcube/Baymax" == 1.1.0
+github "3sidedcube/ThunderTable" == 1.6.0
+github "3sidedcube/ThunderCollection" == 1.3.0
+github "3sidedcube/ThunderBasics" "release/v2.0.0"
```
Comment (BenCShutt) [inline Cartfile:3]: This is a TODO

File: ThunderCloud/EditLocalisationTableViewCell.swift [hunk 2]
```diff
@@ -27,23 +27,23 @@ class EditLocalisationTableViewCell: InputTextViewCell {
...
 		contentView.backgroundColor = .clear
 		backgroundColor = .clear
 		
-		backgroundView = TSCView()
+		backgroundView = UIView()
 		backgroundView?.backgroundColor = .white
-		backgroundView?.borderWidth = 1/UIScreen.main.scale
-		backgroundView?.borderColor = UIColor(hexString: "9B9B9B")
-		backgroundView?.cornerRadius = 2.0
+        backgroundView?.layer.borderWidth = 1/UIScreen.main.scale
```
Comment (simonmitchell) [inline ThunderCloud/EditLocalisationTableViewCell.swift:39]: Can we re-indent this file quickly :D

File: ThunderCloud/ImageSelectionCollectionViewCell.swift [hunk 3]
```diff
@@ -72,7 +72,7 @@ class ImageSelectionCollectionViewCell: UICollectionViewCell {
 	
 	override func prepareForReuse() {
 		super.prepareForReuse()
-		imageView.borderWidth = 0
+        imageView.layer.borderWidth = 0
```
Comment (simonmitchell) [inline ThunderCloud/ImageSelectionCollectionViewCell.swift:75]: Same here, re-indent please :)

File: ThunderCloud/LocalisationController.swift [hunk 4]
```diff
@@ -440,10 +440,10 @@ public class LocalisationController: NSObject {
 	
 	private func addHighlight(to view: UIView, withLocalisationKey localisationKey: String) {
 		
-		let highlightView = TSCView(frame: view.bounds)
-		highlightView.cornerRadius = 4.0
-		highlightView.borderColor = .black
-		highlightView.borderWidth = 1.0
+		let highlightView = UIView(frame: view.bounds)
+        highlightView.layer.cornerRadius = 4.0
```
Comment (simonmitchell) [inline ThunderCloud/LocalisationController.swift:444]: And here :P

File: ThunderCloud/StormLoginViewController.swift [hunk 5]
```diff
@@ -74,8 +78,8 @@ class StormLoginViewController: UIViewController {
 		reasonLabel.text = loginReason ?? "Log into your Storm account"
 		reasonLabel.textColor = UIColor(hexString: "818181")
 		
-		usernameField.borderWidth = 1.0/UIScreen.main.scale
-		passwordField.borderWidth = 1.0/UIScreen.main.scale
+        configure(textField: usernameField)
```
Comment (simonmitchell) [inline ThunderCloud/StormLoginViewController.swift:81]: Re-indent too :D

**Review by simonmitchell — APPROVED** [review]

File: ThunderCloud/UnorderedListItemCell.swift [hunk 6]
```diff
@@ -18,7 +18,11 @@ open class UnorderedListItemCell: StormTableViewCell {
 	@IBOutlet weak private var bulletAlignTopConstraint: NSLayoutConstraint?
 	
 	/// The view which represents the bullet point
-	@IBOutlet weak var bulletView: TSCView!
+    @IBOutlet weak var bulletView: UIView! {
```
Comment (BenCShutt) [inline ThunderCloud/UnorderedListItemCell.swift:21]: Possibly re-indent

**Pull request merged**